cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -87,6 +96,32 @@ export class MigrationWave {
         click(confirmButton);
     }
 
+    public setApplications(toBeCanceled = false): void {
+        if (!this.applications || !this.applications.length) {
+            return;
+        }
+
+        MigrationWave.open();
+        this.expandActionsMenu();
+        cy.contains("Manage applications").click();
```

Comment:
"Manage applications" can be added to constants and can be reused if needed in future.